server/src/test/java/org/eclipse/openvsx/adapter/VSCodeAPITest.java

```diff
@@ -724,9 +751,19 @@ private String file(String name) throws UnsupportedEncodingException, IOExceptio
             return CharStreams.toString(new InputStreamReader(stream, "UTF-8"));
         }
     }
-    
+
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.

Reply:
Just tested this. Please add `SimpleMeterRegistry.class` to the `@MockBean` list and remove the `UrlConfigService` and `MeterRegistry` `@Bean` methods.

Reply:
@amvanbaren should we do the same with the other places you commented this on or just in the VS Code API test?

Reply:
I changed it in all affected files for now, can revert.